packages/kbn-config/src/deprecation/deprecation_factory.ts

```diff
@@ -33,10 +42,18 @@ const _rename = (
   const newValue = get(config, fullNewPath);
   if (newValue === undefined) {
     addDeprecation({
-      message: `"${fullOldPath}" is deprecated and has been replaced by "${fullNewPath}"`,
+      title: getDeprecationTitle(fullOldPath),
+      message: i18n.translate('kbnConfig.deprecations.replacedSettingMessage', {
+        defaultMessage: `Setting "{fullOldPath}" is deprecated and has been replaced by "{fullNewPath}"`,
```

Comment:
I'd avoid using quotes for emphasis (if these are indeed rendered as quotes in the Upgrade Assistant.)

```suggestion
        defaultMessage: `Use {fullNewPath} instead.`,
```

Reply:
We are wrapping all configuration paths in quotations to make sure readers understand the notation. We are doing this consistently all over the deprecations. Do you think we should not do that? My personal opinion is that we should keep them.

Reply:
Personally, I think it's clear they are settings from the context and dot notation and using quotes for emphasis hurts readability. That said, if they're used consistently across all of the deprecations, fiddling around with undoing that probably isn't worth the effort. :-)